compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1406,3 +1406,6 @@ error.methods.not.allowed=\
 
 error.distinct.typing.only.support.objects.and.errors=\
   distinct typing is only supported for object type and error type
+
+warning.compiler.plugin.crashed=\
+  compiler plugin crashed
```

Comment:
Is there a way to identify a plugin (a name perhaps)?
If so, can we print? 
`compiler plugin crashed: xyz plugin`

Reply:
I'm afraid compiler plugins at the moment doesn't have anything like that.
The only way to see which plugin failed it to check the ballerina-internal-log.

We can parse the stack trace to detect which compiler plugin crashed. I think we can do something like that.